### DISCUSSION TITLE
Values for PVStatus in wiki are off

### DISCUSSION THREAD

**Discussion by syntacrsc** [johanmeijer/grott#267] [Q&A]
Hi all.
The Wiki description for the MQTT values for "pvstatus" seem to be off - I feel it might be worth wile updating the wiki entry?!

Wiki simply states:
"pvstatus
Inverter run state:
0:waiting, 1:normal, 2:fault"

Here are the values I was able to identify so far (at least for a SPH10000TL3 BH-UP):
pvstatus = meaning:
0 = status 0 (unknown)
1 = Self inspection
2 = status 2 (unknown)
2 = status 3 (unknown)
2 = status 2 (unknown)
2 = status 5 (unknown)
6 = On-Grid
7 = Off-Grid
8 = status 8 (unknown)
9 = status 9 (unknown)

May someone finds this helpful?

Kind regards,
Ralph...

**Comment by johanmeijer** [comment]
I have not found any documentation hat gives more information for the status of a SPH (but als TL-S, TL-X, SPA, MIX, TL3-X). 

This what is stated: 

    0:waiting,
    1:normal,
    3:fault

The SPF documentation gives some more information on the status, but I am not sure you can apply this also to other inverters: 

![image](https://user-images.githubusercontent.com/16508298/212994920-e417822f-51fa-4685-bb6f-fefd566342e7.png)
